BDC/src/data_collector.py

```diff
@@ -0,0 +1,62 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Ruchita Nathani <ruchita.nathani@fau.de>
+import csv
+import requests
```

Comment:
Please add `requests` package to the dependencies in `Pipfile`. You can do so by running `pipenv install requests` in bash.